### DISCUSSION TITLE
Debug output? Or the curious case of no visible output?

### DISCUSSION THREAD

**Discussion by Narrat** [orhun/gpg-tui#135] [General]
Hello,
I have a curious issue and dunno what is the reason. It happened with an previous versions as well as with the current release (thought about giving it a try again with the 0.10.x release).  It may be very likely pebcac but without details hard to tell :D
What is happening? I don't get any output at all with `gpg-tui`. But there is also no error message or whatsoever.
Is there a way to enable a debug log or output?

![gpg-tui](https://github.com/orhun/gpg-tui/assets/2787518/1da1aa7e-db8e-4d8f-9399-06c2ab12366d)
Exemplary picture. Only thing I get to see, whatever the size of the terminal window. How can I pinpoint the issue?
Is there even an issue? Maybe the first startup takes some time and I'm being really impatient?

**Comment by orhun** [comment]
Hello! 🐻 

It is an error likely related to your GnuPG setup so I'd suggest restarting the `gpg-agent`:

```sh
$ gpgconf --kill gpg-agent
```

About the actual question, I might set up logging similar to [this](https://github.com/orhun/systeroid/#verbose-logging). Can you create an issue to track this?

**Reply by Narrat** [reply]
> It is an error likely related to your GnuPG setup so I'd suggest restarting the `gpg-agent`:
> 
> ```shell
> $ gpgconf --kill gpg-agent
> ```

Sadly that doesn't change a thing.
But I noticed something.
Disclaimer: I'm not very proficient with gpg, but this keyring in general is really old (like 10 years+). There may some stupidities be done :D
I noticed the following output: 
```
$ gpg --list-keys
gpg: key "<fingerprint>" not found: Not found
(...)
```
I do currently not know if this was a deliberate choice (while following some guide) to remove the private key while keeping a signing key or whatever.
Does maybe `gpg-tui` stumble over this?

> About the actual question, I might set up logging similar to [this](https://github.com/orhun/systeroid/#verbose-logging). Can you create an issue to track this?

I created #136 :)

**Reply by orhun** [reply]
> Does maybe gpg-tui stumble over this?

It is probable. You can try running the [list example from gpgme](https://github.com/gpg-rs/gpgme/blob/master/examples/keylist.rs) to check if it working.

> I created https://github.com/orhun/gpg-tui/issues/136 :)

Thank you!

**Reply by Narrat** [reply]
> It is probable. You can try running the [list example from gpgme](https://github.com/gpg-rs/gpgme/blob/master/examples/keylist.rs) to check if it working.

The example works fine and throw no errors.
